unknown type name 'BOOL'

I got below errors:
```
../Lib/cellular/modules/sara_r4/cellular_r4.h:117:5: error: unknown type name 'BOOL'
../Lib/cellular/modules/sara_r4/cellular_r4.h:128:5: error: unknown type name 'BOOL'
../Lib/cellular/modules/sara_r4/cellular_r4.h:129:5: error: unknown type name 'BOOL'
../Lib/cellular/modules/sara_r4/cellular_r4_api.c:1053:72: error: 'TRUE' undeclared (first use in this function)
../Lib/cellular/modules/sara_r4/cellular_r4_api.c:1260:97: error: 'TRUE' undeclared (first use in this function)
../Lib/cellular/modules/sara_r4/cellular_r4_api.c:1419:68: error: 'FALSE' undeclared (first use in this function)
../Lib/cellular/modules/sara_r4/cellular_r4_api.c:2435:94: error: 'TRUE' undeclared (first use in this function)
```

--- a/modules/sara_r4/cellular_r4.h
+++ b/modules/sara_r4/cellular_r4.h
@@ -114,7 +114,7 @@ typedef enum MNOProfileType
  */
 typedef struct CellularPdnContextInfo
 {
-    BOOL contextsPresent[ MAX_PDP_CONTEXTS ];                             /**< Context present in +CGDCONT response or not. */
+    bool contextsPresent[ MAX_PDP_CONTEXTS ];                             /**< Context present in +CGDCONT response or not. */
     char ipType[ MAX_PDP_CONTEXTS ][ CELULAR_PDN_CONTEXT_TYPE_MAX_SIZE ]; /**< PDN Context type. */
     char apnName[ MAX_PDP_CONTEXTS ][ CELLULAR_APN_MAX_SIZE ];            /**< APN name. */
     char ipAddress[ MAX_PDP_CONTEXTS ][ CELLULAR_IP_ADDRESS_MAX_SIZE ];   /**< IP address. */
@@ -125,8 +125,8 @@ typedef struct CellularPdnContextInfo
  */
 typedef struct CellularPdnContextActInfo
 {
-    BOOL contextsPresent[ MAX_PDP_CONTEXTS ]; /**< Context present in +CGACT response or not. */
-    BOOL contextActState[ MAX_PDP_CONTEXTS ]; /**< Context active state from +CGACT response. */
+    bool contextsPresent[ MAX_PDP_CONTEXTS ]; /**< Context present in +CGACT response or not. */
+    bool contextActState[ MAX_PDP_CONTEXTS ]; /**< Context active state from +CGACT response. */
 } CellularPdnContextActInfo_t;
 
 /*-----------------------------------------------------------*/

--- a/modules/sara_r4/cellular_r4_api.c
+++ b/modules/sara_r4/cellular_r4_api.c
@@ -1776,8 +1776,8 @@ static bool _parseExtendedSignalQuality( char * pQcsqPayload,
         {
             /*
              * Bit Error Rate (BER):
-             *  ・0..7: as RXQUAL values in the table in 3GPP TS 45.008 [124], subclause 8.2.4
-             *  ・99: not known or not detectable
+             *  0..7: as RXQUAL values in the table in 3GPP TS 45.008 [124], subclause 8.2.4
+             *  99: not known or not detectable
              */
             if( ( tempValue >= 0 ) && ( tempValue <= 7 ) )
             {
@@ -1821,10 +1821,10 @@ static bool _parseExtendedSignalQuality( char * pQcsqPayload,
         {
             /*
              * Reference Signal Received Quality (RSRQ):
-             *  ・0: less than -19.5 dB
-             *  ・1..33: from -19.5 dB to -3.5 dB with 0.5 dB steps
-             *  ・34: -3 dB or greater
-             *  ・255: not known or not detectable
+             *  0: less than -19.5 dB
+             *  1..33: from -19.5 dB to -3.5 dB with 0.5 dB steps
+             *  34: -3 dB or greater
+             *  255: not known or not detectable
              */
             if( ( tempValue >= 0 ) && ( tempValue <= 34 ) )
             {
@@ -1856,10 +1856,10 @@ static bool _parseExtendedSignalQuality( char * pQcsqPayload,
         {
             /*
              * Reference Signal Received Power(RSRP) :
-             *  ・0 : less than - 140 dBm
-             *  ・1..96 : from - 140 dBm to - 45 dBm with 1 dBm steps
-             *  ・97 : -44 dBm or greater
-             *  ・255 : not known or not detectable
+             *  0 : less than - 140 dBm
+             *  1..96 : from - 140 dBm to - 45 dBm with 1 dBm steps
+             *  97 : -44 dBm or greater
+             *  255 : not known or not detectable
              */
             if( ( tempValue >= 0 ) && ( tempValue <= 97 ) )
             {

--- a/modules/sara_r4/cellular_r4_api.c
+++ b/modules/sara_r4/cellular_r4_api.c
@@ -1776,8 +1776,8 @@ static bool _parseExtendedSignalQuality( char * pQcsqPayload,
         {
             /*
              * Bit Error Rate (BER):
-             *  ・0..7: as RXQUAL values in the table in 3GPP TS 45.008 [124], subclause 8.2.4
-             *  ・99: not known or not detectable
+             *  繝ｻ0..7: as RXQUAL values in the table in 3GPP TS 45.008 [124], subclause 8.2.4
+             *  繝ｻ99: not known or not detectable
              */
             if( ( tempValue >= 0 ) && ( tempValue <= 7 ) )
             {
@@ -1821,10 +1821,10 @@ static bool _parseExtendedSignalQuality( char * pQcsqPayload,
         {
             /*
              * Reference Signal Received Quality (RSRQ):
-             *  ・0: less than -19.5 dB
-             *  ・1..33: from -19.5 dB to -3.5 dB with 0.5 dB steps
-             *  ・34: -3 dB or greater
-             *  ・255: not known or not detectable
+             *  繝ｻ0: less than -19.5 dB
+             *  繝ｻ1..33: from -19.5 dB to -3.5 dB with 0.5 dB steps
+             *  繝ｻ34: -3 dB or greater
+             *  繝ｻ255: not known or not detectable
              */
             if( ( tempValue >= 0 ) && ( tempValue <= 34 ) )
             {
@@ -1856,10 +1856,10 @@ static bool _parseExtendedSignalQuality( char * pQcsqPayload,
         {
             /*
              * Reference Signal Received Power(RSRP) :
-             *  ・0 : less than - 140 dBm
-             *  ・1..96 : from - 140 dBm to - 45 dBm with 1 dBm steps
-             *  ・97 : -44 dBm or greater
-             *  ・255 : not known or not detectable
+             *  繝ｻ0 : less than - 140 dBm
+             *  繝ｻ1..96 : from - 140 dBm to - 45 dBm with 1 dBm steps
+             *  繝ｻ97 : -44 dBm or greater
+             *  繝ｻ255 : not known or not detectable
              */
             if( ( tempValue >= 0 ) && ( tempValue <= 97 ) )
             {

--- a/modules/sara_r4/cellular_r4_api.c
+++ b/modules/sara_r4/cellular_r4_api.c
@@ -1050,7 +1050,7 @@ CellularError_t Cellular_DeactivatePdn( CellularHandle_t cellularHandle,
             }
 
             /* Deactivate context if active */
-            if( pdpContextsActInfo.contextActState[ contextId - 1 ] == TRUE )
+            if( pdpContextsActInfo.contextActState[ contextId - 1 ] == true )
             {
                 /* Don't deactivate LTE default bearer context */
                 /* Otherwise sending AT command "+CGACT=0,1" for deactivation will result in ERROR */
@@ -1257,7 +1257,7 @@ static CellularPktStatus_t _Cellular_RecvFuncGetPdpContextActState( CellularCont
                                         ( tempValue <= ( int32_t ) MAX_PDP_CONTEXTS ) )
                                     {
                                         contextId = ( uint8_t ) tempValue;
-                                        pPDPContextsActInfo->contextsPresent[ contextId - 1 ] = TRUE;
+                                        pPDPContextsActInfo->contextsPresent[ contextId - 1 ] = true;
                                         CellularLogDebug( "_Cellular_RecvFuncGetPdpContextActState: Context Id: %d", contextId );
                                     }
                                     else
@@ -1416,7 +1416,7 @@ CellularError_t Cellular_ActivatePdn( CellularHandle_t cellularHandle,
         }
 
         /* Activate context if not already active */
-        if( pdpContextsActInfo.contextActState[ contextId - 1 ] == FALSE )
+        if( pdpContextsActInfo.contextActState[ contextId - 1 ] == false )
         {
             if( pktStatus == CELLULAR_PKT_STATUS_OK )
             {
@@ -1776,8 +1776,8 @@ static bool _parseExtendedSignalQuality( char * pQcsqPayload,
         {
             /*
              * Bit Error Rate (BER):
-             *  • 0..7: as RXQUAL values in the table in 3GPP TS 45.008 [124], subclause 8.2.4
-             *  • 99: not known or not detectable
+             *  E0..7: as RXQUAL values in the table in 3GPP TS 45.008 [124], subclause 8.2.4
+             *  E99: not known or not detectable
              */
             if( ( tempValue >= 0 ) && ( tempValue <= 7 ) )
             {
@@ -1821,10 +1821,10 @@ static bool _parseExtendedSignalQuality( char * pQcsqPayload,
         {
             /*
              * Reference Signal Received Quality (RSRQ):
-             *  • 0: less than -19.5 dB
-             *  • 1..33: from -19.5 dB to -3.5 dB with 0.5 dB steps
-             *  • 34: -3 dB or greater
-             *  • 255: not known or not detectable
+             *  E0: less than -19.5 dB
+             *  E1..33: from -19.5 dB to -3.5 dB with 0.5 dB steps
+             *  E34: -3 dB or greater
+             *  E255: not known or not detectable
              */
             if( ( tempValue >= 0 ) && ( tempValue <= 34 ) )
             {
@@ -1856,10 +1856,10 @@ static bool _parseExtendedSignalQuality( char * pQcsqPayload,
         {
             /*
              * Reference Signal Received Power(RSRP) :
-             *  • 0 : less than - 140 dBm
-             *  • 1..96 : from - 140 dBm to - 45 dBm with 1 dBm steps
-             *  • 97 : -44 dBm or greater
-             *  • 255 : not known or not detectable
+             *  E0 : less than - 140 dBm
+             *  E1..96 : from - 140 dBm to - 45 dBm with 1 dBm steps
+             *  E97 : -44 dBm or greater
+             *  E255 : not known or not detectable
              */
             if( ( tempValue >= 0 ) && ( tempValue <= 97 ) )
             {
@@ -2432,7 +2432,7 @@ static CellularPktStatus_t _Cellular_RecvFuncGetPdpContextSettings( CellularCont
                                         ( tempValue <= ( int32_t ) MAX_PDP_CONTEXTS ) )
                                     {
                                         contextId = ( uint8_t ) tempValue;
-                                        pPDPContextsInfo->contextsPresent[ contextId - 1 ] = TRUE;
+                                        pPDPContextsInfo->contextsPresent[ contextId - 1 ] = true;
                                         CellularLogDebug( "_Cellular_RecvFuncGetPdpContextSettings: Context Id: %d", contextId );
                                     }
                                     else

